### DISCUSSION TITLE
V3.lua can not work

### DISCUSSION THREAD

**Discussion by sanmaodeai** [ExpressLRS/ExpressLRS#1895] [Q&A]
My radio is frsky X-LITE PRO ,  I update the firmware of OPENTX V2.3.5 , and I set the external RF with CRSF . But I run the V3 lua ,it show the error with ( Syntax error elrsV3.lua:1 unexpected symbol ),Does anyone else have this problem?

**Comment by JyeSmith** [comment]
OTX V2.3.5 this is now over 2yrs old and probably missing some font/symbols.

I would recommend updating to EdgeTX 2.7 and trying again.

**Reply by sanmaodeai** [reply]
thanks for your suggestion,but I don't know whether my radio support the EdgeTX .

**Reply by JyeSmith** [reply]
It is supported and updating is pretty easy.
